bukkit/src/main/java/net/william278/husktowns/hook/EconomyHook.java

```diff
@@ -1,5 +1,6 @@
 package net.william278.husktowns.hook;
 
+import dev.unnm3d.rediseconomy.api.RedisEconomyAPI;
```

Comment:
This breaks the Vault economy support. The plugin will crash with a ClassDefNotFoundException as you're trying to import a class that may not exist.